wondrous-app/components/Common/Task/Task.tsx

```diff
@@ -263,7 +245,8 @@ export function Task(props) {
         },
       ]);
     } else {
-      setUserList(users);
+      // TODO: many users in one task? there is no such thing as 'users' field in task
+      setUserList([]);
```

Comment:
setUserList sets the assignee I believe

Reply:
yeah this is poorly named

Reply:
I mean, why we even use effect here, it had `users` in props, but there is no such field as `users` in task

Reply:
I've checked graphql api on frontend, and haven't found anything about it

Reply:
I think this is to ensure assignee is updated after we create an assignee